python/ray/dashboard/modules/node/tests/test_node.py

```diff
@@ -191,40 +208,44 @@ def test_multi_node_churn(
     assert wait_until_server_available(cluster.webui_url) is True
     webui_url = format_web_url(cluster.webui_url)
 
```

Comment:
`test_multi_node_churn` is the test that's supposed to catch the bug but it turned out it's broken and always passes:

`cluster.add_node()` cannot run in non-main thread and it will throw exception so the `cluster_chaos_monkey` thread will exit immediately without adding or removing any nodes. This PR swaps the code of main thread and the other thread.